Source/Core/Editor/TextToSpeech/TextToSpeechBuildPreprocessor.cs

```diff
@@ -9,14 +18,37 @@ namespace VRBuilder.Core.Editor.TextToSpeech
     /// </summary>
     public class TextToSpeechBuildPreprocessor : IPreprocessBuildWithReport
     {
-        public int callbackOrder => 1;
+        public int callbackOrder => 0;
 
-        /// <summary>
-        /// Generates TTS files for all processes before a build.
-        /// </summary>
         public void OnPreprocessBuild(BuildReport report)
         {
-            TextToSpeechEditorUtils.GenerateTextToSpeechForAllProcesses();
+            if (TextToSpeechSettings.Instance.GenerateAudioInBuildingProcess)
+            {
+                Task task = TextToSpeechEditorUtils.GenerateTextToSpeechForAllProcesses();
+                task.Wait();
+                
+                //TODO enable TTS building for active scenes only
```

Comment:
Is it worth keeping this TODO and commented out code? Please consider removing it.